docs/app/Examples/modules/Checkbox/States/Disabled.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import {Form, Checkbox} from 'stardust';
+
+export default class extends Component {
+  render() {
+    return (
+      <div>
+        <Form>
+          <Checkbox className='disabled' label='Disabled' />
+        </Form>
+        <Form>
+          <Checkbox className='toggle' disabled={true} label='Disabled' />
+        </Form>
```

Comment:
Use a single `Form` and wrap the `Checkbox` components in `Field` components to get the proper whitespace around them:

![image](https://cloud.githubusercontent.com/assets/5067638/10502001/16630f9a-729c-11e5-86e8-d29d057ec378.png)

vs

![image](https://cloud.githubusercontent.com/assets/5067638/10502018/4b979c3a-729c-11e5-96e6-7eb431122ed6.png)